setup.py

```diff
@@ -26,7 +23,8 @@
         'cartopy',
         'cdo',
         'cdsapi',
-        'cf-units',
+        # see https://github.com/SciTools/cf-units/issues/218
+        'cf-units>=3.0.0,!=3.0.1.post0',
```

Comment:
Does this need to match ESMValCore? 

`'cf-units>=3.0.0,<3.1.0,!=3.0.1.post0',`

Reply:
Done